src/registrar/templates/includes/profile_information.html

```diff
@@ -0,0 +1,36 @@
+{% load static field_helpers %}
+
+{% block domain_content %} 
+
+<!-- {% if not user.has_contact_info %}
```

Comment:
Q: Placeholder for a future ticket?

Reply:
Seems like we would need *something* for error handling.  But it wasn't specified in the scope of this ticket.  I scaffolded this to help me debug data flows an figured....maybe we could leave it...

Reply:
I do not think it's needed though, with the contact-user collapse that we have planned. I suggest just removing this.